### PR TITLE
change version to release the harvester v1.0.3 plugin

### LINKED ISSUE #6856: Harvester Plugin: Release 1.0.3
- remove `-head` from harvester package.json version in `release-harvester-plugin-v1.0.3` branch
- ensure a build successfully completes in drone
- See confluence for details

### PR DESCRIPTION
**Pull request by WuJun2016**
### Summary
change v1.0.3 plugin version

Fixes #6856

### CODE REVIEW

**Comment by WuJun2016** [comment]
Hi， @mantis-toboggan-md    The harvester QA team will be testing the v1.0.3 plugin next week, so this pr will not be merged for now, it just needs to be merged before the official v2.6.9 release of rancher.

**Comment by mantis-toboggan-md** [comment]
I understand. We wont merge until right before the 2.6.9 release and our release captain is aware of that. There is also an issue associated with this: #6856

**Review by richard-cox — APPROVED** [review]

**Pull request merged**
